modules/openapi/component-protocol/scenarios/action/components/actionForm/render.go

```diff
@@ -330,12 +330,32 @@ func registerActionTypeRender() {
 				return nil
 			}
 
-			if c.Props != nil {
+			defer func() {
+				if c.Props == nil {
+					return
+				}
+
 				value, ok := c.Props.(map[string]interface{})
-				if ok {
-					value["scopeID"] = bdl.InParams["scopeID"]
+				if !ok {
+					return
 				}
-			}
+
+				fields, ok := value["fields"].([]apistructs.FormPropItem)
+				if !ok {
+					return
+				}
+
+				for _, field := range fields {
+					if field.Component != "memberSelector" {
+						continue
+					}
+					props, ok := field.ComponentProps.(map[string]interface{})
+					if !ok {
+						continue
+					}
+					props["scopeId"] = bdl.InParams["appId"]
```

Comment:
why change `scopeID` to `scopeId` ?

Reply:
In order to meet the requirements of front-end componentization